Release notes for WAN Assurance plugin 3.7.7 (#678)
128technology/docs: docs/release_notes_wan_assurance_plugin_3.7.md

title: WAN Assurance Plugin 3.7 Release Notes
sidebar_label: '3.7'
---
## Release 3.7.7

> **Release Date:** TBD

Reviewer: - [ ] Add release date

- **WAN-2518 _internal_ network showing up on the Networks dropdown for Application Policy Insights**

  _**Resolution**_ The internal network will not be included in the data reported for the feature.

- **WAN-2478 Disabling WAN Assurance plugin does not stop the mist-agent service**

  _**Resolution**_ When the WAN Assurance plugin is disabled on a router, all relevant services are now stopped.

- **WAN-2678 MIST onboarding for HA routers fails with a `mutual authentication failed` error**

  _**Resolution**_ The condition that checks for the completeness of the HA configuration has been made more robust, preventing an authentication failure error. This allows the completion of the MIST onboarding process.

- **WAN-2754 No peer path up events seen for interfaces on the Insights page**

  _**Resolution**_ The handling of cached peer path events has been corrected to no longer crash.

- **WAN-2748 MIST org sees too many LTE ARP interface flaps**

  _**Resolution**_ ARP event generation for bad LTE connections has been made more robust to prevent false positives.

> - **WAN-2858 LLDP client information not send for routers running version 6.2.3 software**

Reviewer: typo: "not sent"

Author: done

title: WAN Assurance Plugin 3.7 Release Notes
sidebar_label: '3.7'
---
## Release 3.7.7

**Release Date:** March 29, 2024

### Resolved Issues

- **WAN-2121 Port down events are not seen on the Insights page after a cable disconnect**

  _**Resolution**_ When a cable was disconnected on an interface with an active MIST connection, some events were getting dropped. The mechanism is now more robust and handles the reconnect scenario more gracefully.

> - **WAN-2240 WAN bandwidth graph is blank after the user adds new WAN interfaces: oc-stats from the SSR is missing some interfaces**

Reviewer 2: ```suggestion
- **WAN-2240 WAN bandwidth graph is blank after the user adds new WAN interfaces: `oc-stats` from the SSR is missing some interfaces**
```

  _**Resolution**_ The logic to include configured interfaces in the `oc-stats` and validating data for completeness is more robust, ensuring that accurate data is sent for new configuration changes.

> - **WAN-2478 Disabling WAN Assurance plugin does not stop the mist-agent service**

Reviewer 2: ```suggestion
- **WAN-2478 Disabling the WAN Assurance plugin does not stop the mist-agent service**
```

  _**Resolution**_ When the WAN Assurance plugin is disabled on a router, all relevant services are now stopped.

- **WAN-2518 _internal_ network showing up on the Networks dropdown for Application Policy Insights**

  _**Resolution**_ The internal network will not be included in the data reported for the feature.

- **WAN-2637 One of the nodes in HA pair was not sending any data to the cloud**

>   _**Resolution**_ The runtime par file corruption of monitoring-agent package is being addressed by adopting a better packaging approach.

Reviewer 2: ```suggestion
  _**Resolution**_ The runtime par file corruption of the monitoring-agent package has been addressed by adopting a better packaging approach.
```
Also, is the "runtime par file" correct? I've not heard of a "par" file. Just verifying that it's not a typo.

- **WAN-2647 Peer path status is displayed incorrectly under Topology Details view for HA routers**

  _**Resolution**_ For VRRP based redundancy, the active/standby status will be reported correctly.

- **WAN-2678 MIST onboarding for HA routers fails with a `mutual authentication failed` error**

  _**Resolution**_ The condition that checks for the completeness of the HA configuration has been made more robust, preventing an authentication failure error. This allows the completion of the MIST onboarding process.

> - **WAN-2696 Client information displays user as `Anonymous` **

Reviewer 2: ```suggestion
- **WAN-2696 Client information displays user as `Anonymous`**
```

  _**Resolution**_ In addition to the realtime lease updates, the full DHCP server lease data is sent to the cloud once a day.

- **WAN-2748 MIST org sees too many LTE ARP interface flaps**

  _**Resolution**_ ARP event generation for bad LTE connections has been made more robust to prevent false positives.

- **WAN-2754 No peer path up events seen for interfaces on the Insights page**

  _**Resolution**_ The handling of cached peer path events has been corrected to no longer crash.

- **WAN-2858 LLDP client information not sent for routers running version 6.2.3 software**

  _**Resolution**_ The logic to check for `LLDP enabled` now covers the new mechanism used in 6.2.3 software allowing the LLDP client information to be successfully sent to the MIST cloud.

- **WAN-2861 LTE_stats are received from the device but are missing in the GUI**

  _**Resolution**_ LTE devices with no peer paths were incorrectly being reported as a LAN interface. LTE interfaces will now always be reported as WAN interfaces.

> - **WAN-2897 Router not reporting CPU, memory and port metrics**

Reviewer 2: ```suggestion
- **WAN-2897 Router not reporting CPU, memory, and port metrics**
```
